### ISSUE TITLE
Error in console UI after deploying the operator helm chart: `open build/index.html: file does not exist`

### ISSUE THREAD

**Issue by usersina** [minio/operator#1528]
I installed the MinIO operator using the [Helm Chart](https://github.com/minio/operator/tree/master/helm/tenant) with the default configuration, port forwarded the console to access it in a browser, but I see the following error instead:

> open build/index.html: file does not exist

## Expected Behavior
The MinIO console should successfully show up in http://localhost:9090 after a default Helm Chart installation and a port forward.

## Current Behavior
Browser shows the following output
> open build/index.html: file does not exist

Here is a more detailed curl output

```bash
bash-4.4$ curl -v localhost:9090

* Rebuilt URL to: localhost:9090/
*   Trying ::1...
* TCP_NODELAY set
* Connected to localhost (::1) port 9090 (#0)
> GET / HTTP/1.1
> Host: localhost:9090
> User-Agent: curl/7.61.1
> Accept: */*
>
< HTTP/1.1 500 Internal Server Error
< Content-Type: text/plain; charset=utf-8
< Server: MinIO Operator
< Vary: Accept-Encoding
< X-Content-Type-Options: nosniff
< X-Frame-Options: DENY
< X-Xss-Protection: 1; mode=block
< Date: Tue, 28 Mar 2023 09:06:55 GMT
< Content-Length: 43
<
open build/index.html: file does not exist
* Connection #0 to host localhost left intact
```

## Steps to Reproduce

Installing the Helm chart

1. Configure the repo
```bash
helm repo add minio https://operator.min.io/
```

2. Install the chart
```bash
helm upgrade \
    --install \
    --namespace minio-operator \
    --create-namespace \
    minio-operator minio/operator
```

3. Port forward the console

```bash
kubectl --namespace minio-operator port-forward svc/console 9090:9090 --address='0.0.0.0'
```

4. Access the console in a browser or with curl

```bash
curl -v localhost:9090
```

## Your Environment
* Version used (`minio-operator`): `v5.0.0` (default from the Chart)
* Environment name and version: `K8s Client Version: v1.26.3` | `K8s Server Version: v1.26.1`
* Operating System and version (`uname -a`): `Linux kube-controller 5.4.0-144-generic #161-Ubuntu SMP Fri Feb 3 14:49:04 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux`

**Comment by sylvainOL** [comment]
Hello, I've got the exact same issue here as well

**Comment by dvaldivia** [comment]
That's strange, can you confirm the image the console deployment on the MinIO-operator namespace is using?

**Comment by usersina** [comment]
@dvaldivia, it's the same as the one specified in the [values.yaml](https://github.com/minio/operator/blob/master/helm/operator/values.yaml) of the chart

```bash
kubectl describe deployment console -n minio-operator
```

```yml
Pod Template:
  Labels:           app.kubernetes.io/instance=minio-operator-console
                    app.kubernetes.io/name=operator
  Service Account:  console-sa
  Containers:
   operator:
    Image:       quay.io/minio/operator:v5.0.0 # <==
    Ports:       9090/TCP, 9443/TCP
    Host Ports:  0/TCP, 0/TCP
```

**Comment by sylvainOL** [comment]
hey, same for me: 
```
quay.io/minio/operator:v5.0.0
```

**Comment by sylvainOL** [comment]
If we look at [Dockerfile](https://github.com/minio/operator/blob/master/Dockerfile), there are no `build/index.html` added and there are no also when running:

```console
bash-4.4$ ls -lh
total 81M
lrwxrwxrwx   1 root root     7 Jun 21  2021 bin -> usr/bin
dr-xr-xr-x   2 root root     6 Jun 21  2021 boot
drwxr-xr-x   5 root root   360 Mar 28 09:29 dev
drwxr-xr-x   1 root root    92 Mar 28 06:53 etc
drwxr-xr-x   2 root root     6 Jun 21  2021 home
lrwxrwxrwx   1 root root     7 Jun 21  2021 lib -> usr/lib
lrwxrwxrwx   1 root root     9 Jun 21  2021 lib64 -> usr/lib64
drwxr-xr-x   1 root root    21 Mar 28 06:53 licenses
drwx------   2 root root     6 Mar 22 11:19 lost+found
drwxr-xr-x   2 root root     6 Jun 21  2021 media
-rwxrwxr-x   1 root root   81M Mar 28 06:53 minio-operator
drwxr-xr-x   2 root root     6 Jun 21  2021 mnt
drwxr-xr-x   2 root root     6 Jun 21  2021 opt
dr-xr-xr-x 617 root root     0 Mar 28 09:29 proc
dr-xr-x---   3 root root  4.0K Mar 22 11:37 root
drwxr-xr-x   1 root root    21 Mar 22 11:37 run
lrwxrwxrwx   1 root root     8 Jun 21  2021 sbin -> usr/sbin
drwxr-xr-x   2 root root     6 Jun 21  2021 srv
dr-xr-xr-x  13 root root     0 Mar 28 09:29 sys
drwxrwsrwx   3 root 10000   19 Mar 28 09:29 tmp
drwxr-xr-x   1 root root    81 Mar 22 11:19 usr
drwxr-xr-x   1 root root    41 Mar 22 11:20 var
```

maybe it's missing?

**Comment by GenLN** [comment]
Same issue here with operator console... using port-forward for console service on port 9090
tried images from dockerhub and quay.io

**Comment by GenLN** [comment]
minio-console works as i port-forward it, it's just operator console affected

**Comment by pjuarezd** [comment]
Fixed in release 5.0.1
